### ISSUE TITLE
Keras equivalent of "class_weight" in fit method in Tensorflow Estimator

### ISSUE THREAD

**Issue by midnitekoder** [tensorflow/tensorflow#22184]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
- **Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device**:
- **TensorFlow installed from (source or binary)**:
- **TensorFlow version (use command below)**:
- **Python version**:
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**:

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem
I want to weight down the imbalanced class documents by implementing Keras "class_weight" provided in fit method. Is there any equivalent of it in Tf.estimator?

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by facaiy** [comment]
How about preprocessing your label with [tf.feature_column.weighted_categorical_column](https://www.tensorflow.org/api_docs/python/tf/feature_column/weighted_categorical_column) ?

**Comment by ymodak** [comment]
@midnitekoder Did you try the @facaiy 's  suggestion? Does it solve your issue?

> How about preprocessing your label with [tf.feature_column.weighted_categorical_column](https://www.tensorflow.org/api_docs/python/tf/feature_column/weighted_categorical_column) ?

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
